pynitrokey/cli/nk3/secrets.py

```diff
@@ -0,0 +1,335 @@
+from base64 import b32decode
+from typing import List, Optional
+
+import click
+import fido2
+
+from pynitrokey.cli.nk3 import Context, nk3
+from pynitrokey.helpers import AskUser, local_print
+from pynitrokey.nk3.secrets_app import STRING_TO_KIND, Algorithm, SecretsApp
+
+
+@nk3.group()
+@click.pass_context
+def secrets(ctx: click.Context) -> None:
+    """Nitrokey Secrets App. Manage OTP secrets on the device.
+    Use NITROPY_SECRETS_PASSWORD to pass password for the scripted execution."""
+    pass
+
+
+@secrets.command()
+@click.pass_obj
+@click.argument(
+    "name",
+    type=click.STRING,
+)
+@click.argument(
+    "secret",
+    type=click.STRING,
+    # help="The shared secret string (by default in base32)",
```

Comment:
Why is this commented?

Reply:
Good spot! No clue. Will uncomment.

Reply:
Turns out one cannot describe positional argument with click. Earlier this was an optional, hence the field. It was left to leave some info for the future.